iceoryx_utils/include/iceoryx_utils/internal/concurrent/trigger_queue.inl

```diff
@@ -60,7 +60,9 @@ inline bool TriggerQueue<T, CAPACITY>::blocking_pop(T& out)
 template <typename T, uint64_t CAPACITY>
 inline bool TriggerQueue<T, CAPACITY>::try_pop(T& out)
 {
-    if (!m_semaphore->tryWait())
+    if (!m_semaphore->tryWait()
```

Comment:
I don't really like this. There is too much going on in the condition of the if statement. Older developer like me have already forgotten to be in the if statement when reaching `get_value` ;)
Please decouple to error printing with e.g. an else clause. Terminating because of trying to access an non existing value should also be explicit